components/EditorOverview.vue

```diff
@@ -67,7 +67,22 @@ const { $track } = useNuxtApp();
 function playDemoClicked(){
   demoEnabled.value = true;
 
-  demoCanvas.value?.scrollIntoViewIfNeeded();
+
+/**
+ * Listen to DOM Mutations and wait for editorjs element to be inserted 
+ * then scroll to it. We have to wait for the editorjs element to be inserted
+ * otherwise the scroll will not work
+ */
+  const onDomChange=()=>{
+    if(document.getElementById('editorjs')){
+      smoothScrollToCenter(demoCanvas.value)
+      window.removeEventListener('DOMNodeInserted',onDomChange)
+    }
+  }
+  
+  window.addEventListener('DOMNodeInserted',onDomChange)
```

Comment:
it's better to bind this event inside the `onMounted` hook to prevent SSR issues

Reply:
@neSpecc moved the scroll logic into EditorDemo.vue inside onMounted method

Reply:
I changed it, I placed mobile focus part inside setTimeout because it solves to problems 
1. Virtual keyboard interrupts scroll
2. Editor is focused before it's initialised